### DISCUSSION TITLE
handling GIFs

### DISCUSSION THREAD

**Discussion by mkikets99** [telegraf/telegraf#1601] [Q&A] [answered]
Maybe I missed something from docs.
Currently didn't have time to test how it handles...

How to handle sended gifs from user? Is it `photo` or `video`?
Or better just handle it with `message` and depending on responce just look for the file id?

**Comment by MKRhere** [comment, marked as answer]
https://core.telegram.org/bots/api#animation

### Animation

This object represents an animation file (GIF or H.264/MPEG-4 AVC video without sound).

**Reply by mkikets99** [reply]
so... just writing `bot.on('animation',(ctx)=>{})` must work, isn't it?

**Reply by MKRhere** [reply]
Correct

**Reply by mkikets99** [reply]
Ok, i just started configure a repository for project... I'll get In contact soon
